util/ct_log_list.cnf

```diff
@@ -0,0 +1,9 @@
+# This file specifies the Certificate Transparency logs
+# that are to be trusted.
+
+# This file is empty in the OpenSSL source distribution.  One way
+# to generate a file (such as from Chrome's trust list) is to
+# fetch and run this python script:
+# https://github.com/RJPercival/certificate-transparency/blob/0a753fb4487bbf33de7b202133aac95cd0bf21b2/python/utilities/log_list/openssl_generator.py
+# on the log list found at https://sites.google.com/site/certificatetransparency/known-logs.
```

Comment:
Could you make this link http://www.certificate-transparency.org/known-logs please?